docs/driver-only-builds.md

```diff
@@ -238,37 +240,124 @@ The same holds for the associated algorithm:
 `[PSA_WANT|MBEDTLS_PSA_ACCEL]_ALG_FFDH` allow builds accelerating FFDH and
 removing builtin support (i.e. `MBEDTLS_DHM_C`).
 
-### Limitations
-Support for deterministic derivation of a DH keypair
-(i.e. `PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_DERIVE`) is not supported.
-
-AEADs
------
-
-[This section might contain incomplete data and it is going to be updated in
-#8358, i.e. the wrap-up task for accelerated ciphers and AEADs.]
-
-It is possible to have all AEADs operations provided only by a driver.
-
-More precisely you can:
-- enable desired PSA algorithm(s) and key type(s):
-  - `PSA_WANT_ALG_[CCM|GCM]` with `PSA_WANT_KEY_TYPE_[AES|ARIA|CAMELLIA]`
-  - `PSA_WANT_ALG_CHACHA20_POLY1305` with `PSA_WANT_KEY_TYPE_CHACHA20`;
-- enable `MBEDTLS_PSA_ACCEL_xxx` symbol(s) which correspond to the
-  `PSA_WANT_xxx` of the previous step (both for algorithms and key types);
-- disable builtin support of `MBEDTLS_[CCM|GCM|CHACHAPOLY|POLY1305]_C`
-  algorithms and key types `MBEDTLS_[AES|ARIA|CAMELLIA|CHACHA20]_C` for AEADs
-  which are accelerated.
-
-In a build in which all AEADs algorithms and related key types are accelerated
-all AEADs operations requested through the PSA Crypto API (including those in
-TLS and X.509) will be performed by the driver.
-Moreover if no unauthenticated cipher is required, it is also possible to
-disable all built-in block cipher's key types
-(i.e. `MBEDTLS_[AES|ARIA|CAMELLIA|CHACHA20]_C`) and `MBEDTLS_CIPHER_C`. This
-helps in further reducing code's footprint, but unfortunately it makes the
-following modules unavailable:
-- `MBEDTLS_PKCS[5|12]_C`
-- `MBEDTLS_CTR_DRBG_C`
-- `MBEDTLS_NIST_KW_C`
+Ciphers and AEADs
+-----------------
+
```

Comment:
I believe what's available via `mbedtls_cipher_xxx` is:
- For things not using `mbedtls_cipher_setup_psa()`, exactly what has the legacy symbol enabled (`AES_C`, `CIPHER_MODE_xxx`, `GCM_C`, etc). But at the moment we don't have an actual test build for that - we'd need a test with some things accelerated and some not accelerated, because in the current test everything is accelerated so nothing works in `mbedtls_cipher_`.
- When using `mbedtls_cipher_setup_psa()` there might be a few more things working but I'm not sure. (However the function is deprecated, so perhaps not a priority.

Reply:
So, if I understood correctly, I should wait for the new test component before modifying this section, am I right?

Reply:
Perhaps just leave a TODO for now, then. That is, describe everything we know about other modules, but when it comes to `mbedtls_cipher_` just say "TODO". Or we can be bold and describe what we think should work in such a build, together with a "(TODO: confirm)".